packages/manager/apps/key-management-service/src/data/api/okmsCredential.ts

```diff
@@ -1,20 +1,40 @@
+import { ColumnSort } from '@ovhcloud/manager-components';
+import apiClient from '@ovh-ux/manager-core-api';
+import { OkmsCredential } from '@/types/okmsCredential.type';
+import { sortCredentials } from '@/utils/credential/credentialDatagridSort';
+
 /**
  *  Get okms Credential list
  */
 
-import apiClient from '@ovh-ux/manager-core-api';
-import { OkmsCredential } from '@/types/okmsCredential.type';
-
-export const getOkmsCredentialList = async (
+export const getOkmsCredentials = async (
   okmsId: string,
 ): Promise<{ data: OkmsCredential[] }> => {
   return apiClient.v2.get(`okms/resource/${okmsId}/credential`);
 };
 
-export const getOkmsCredentialListQueryKey = (okmsId: string) => [
+export const getOkmsCredentialsQueryKey = (okmsId: string) => [
   `get/okms/resource/${okmsId}/credential`,
 ];
 
+export const sortOkmscredentials = (
+  credentials: OkmsCredential[],
+  sorting: ColumnSort,
+): OkmsCredential[] => {
+  const data = [...credentials];
+
+  if (sorting) {
+    const { id: sortKey, desc } = sorting;
+
+    data.sort(sortCredentials(sortKey as keyof OkmsCredential));
+    if (desc) {
+      data.reverse();
+    }
+  }
+
+  return data;
+};
+
```

Comment:
On last react meeting, control tower said we don't sort v2 endpoint responses (it should be an api feature)
I don't know if it is a special use case though

Reply:
ApiResponse comes from @ovh-ux/manager-core-api package